doc/API.rst.txt

```diff
@@ -170,6 +172,7 @@ change is made.
    ndimage
    odr
    optimize
+   show_config
```

Comment:
please remove this as well

Reply:
I left the reference of `show_config` and will wait for https://github.com/scipy/scipy/pull/17879 to be merged and rebase. Doc build will fail out of a warning till then

Reply:
It's better to remove all that, because gh-17879 already has everything needed. Also, I'd like to merge this PR first.

Reply:
Ah ok, deleted the commit